pkgs/development/python-modules/matrixzulipbridge/default.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  poetry-core,
+  python3Packages,
+}:
+buildPythonPackage rec {
+  pname = "MatrixZulipBridge";
+  version = "0.4.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "GearKite";
+    repo = "MatrixZulipBridge";
+    rev = "9e023efc024bbc610785fcc486981c5676729bce";
+    hash = "sha256-PbcRS+l4AJbYNssqwVNL2a8fghQ8s56/x/m45gkRwFc=";
+  };
+
+  build-system = [ poetry-core ];
+
+  dependencies = with python3Packages; [
+    zulip
+    beautifulsoup4
+    bidict
+    coloredlogs
+    emoji
+    markdownify
+    mautrix
+    python-dotenv
+    ruamel-yaml
+    zulip-emoji-mapping
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'bidict = "^0.22"' 'bidict = "^0.23"' \
+      --replace 'markdownify = "^0.10"' 'markdownify = "^0.12"' \
+      --replace 'ruamel-yaml = ">=0.16, <0.18"' 'ruamel-yaml = ">=0.16, <0.19"' \
+      --replace 'zulip-emoji-mapping = "^1.0.1"' 'zulip-emoji-mapping = "*"' \
+  '';
+
```

```suggestion

pythonImportsCheck = [
  "matrix-zulip-bridge"
];

```

This confirms that the package built and is importable.